includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -36,7 +36,8 @@ public function getData() {
 					'title' => [
 						'type' => 'text',
 						'value' => 'Fandom powered by Wikia'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "value" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)